src/lollipopChart.js

```diff
@@ -362,7 +388,7 @@ var LollipopChart = function (selection) {
   chart.lollipopRadius = function(_) {
     if(!arguments.length) return lollipopRadius;
     lollipopRadius = _;
-    chartGutter = _;
+    //chartGutter = _; //is this a bug?
```

Comment:
Thank you for catching this, it seems the lollipopRadius setter function doesn't alter the gutter size when called on the data setter function which was causing me confusion:

`lollipopChartIndividualScales
  .data(dataIndividualScales).lollipopRadius(15);`:
<img width="469" alt="screen shot 2017-03-17 at 11 48 14 am" src="https://cloud.githubusercontent.com/assets/3695899/24051196/c33423e4-0b07-11e7-8560-e5af99a730b6.png">


`lollipopChartIndividualScales.lollipopRadius(15)
  .data(dataIndividualScales);`:
<img width="476" alt="screen shot 2017-03-17 at 11 48 48 am" src="https://cloud.githubusercontent.com/assets/3695899/24051203/c74aadf4-0b07-11e7-9f0f-69ede2b36b80.png">